lib/karma-electron-launcher.js

```diff
@@ -1,11 +1,22 @@
 // Load in our dependencies
 // DEV: We use `require`/`assert` instead of `peerDependencies` to make floating dependencies easier
 var electronPrebuilt;
-try {
+
+function moduleExists(moduleName) {
+  try {
+    return require.resolve(moduleName);
+  } catch (err) {
+    return false;
+  }
+}
+
+if (moduleExists('electron-prebuilt')) {
```

Comment:
We should prob require `electron` first in case developers forget to uninstall stale dependencies